### ISSUE TITLE
CleanupStoredMailJob spec fails sometimes

### ISSUE THREAD

**Issue by Matthijsy** [csvalpha/amber-api#82]
Sometimes the spec of the mail cleanup job fails

```

1) CleanupExpiredStoredMailsJob#perform when with both deleted emails and expired emails is expected to have received inform_slack(2, 1) 1 time
--
 expected: (2, 1)
 got: (1, 1)

```

**Comment by Matthijsy** [comment]
@cmitz Might be something with the `and_call_original` of the spy? Of maybe the second before call in the tests, iirc multiple before calls cause weird stuff

**Comment by Matthijsy** [comment]
Fixed in #96
